refinery/data_set_manager/isa_tab_parser.py

```diff
@@ -565,11 +571,14 @@ def _parse_term_information(self, headers, row):
                 row.popleft()
```

Comment:
I'm stepping back and wondering if it'd be more helpful if the error message described the pair of elements it expected to see, rather than just the one. "Unexpected element" doesn't really tell the user what *was* expected.

(The code is also redundant in many ways: `row.popleft()` returns the value popped.)

Reply:
It would probably be more helpful, but I have no clue as to what should be expected. I'd need some outside input. I can mange fixing some of the redundancies though